java/src/main/java/ai/rapids/cudf/ParquetOptions.java

```diff
@@ -56,6 +68,41 @@ public Builder withTimeUnit(DType unit) {
       return this;
     }
 
+    /**
+     * Include one or more specific columns.  Any column not included will not be read.
+     * @param names the name of the column, or more than one if you want.
+     */
+    public Builder includeColumn(String... names) {
+      for (String name : names) {
+        includeColumnNames.add(name);
+        binaryAsStringColumns.add(true);
+      }
+      return this;
+    }
+
+    /**
+     * Include this column.
+     * @param name the name of the column
+     * @param isBinary whether this column is to be read in as binary
+     */
+    public Builder includeColumn(String name, boolean isBinary) {
+      includeColumnNames.add(name);
```

Comment:
This is one reason to keep the base class version - can call super(name) here as well if it is there.

Reply:
I'm OK keeping this as-is though.